src/main/scala/nl.knaw.dans.easy.bagindex/server/BagIndexServletComponent.scala

```diff
@@ -86,7 +86,7 @@ trait BagIndexServletComponent {
       }
 
       Option(params)
-        .filter(_.nonEmpty)
+        .filter(_.size > 0)
```

Comment:
why? doesn't `_.nonEmpty` work anymore?

Reply:
This method was deprecated in a newer version of Scalatra, but for this version I reverted the change.